### ISSUE TITLE
Select provider for code examples

### ISSUE THREAD

**Issue by mattheworiordan** [ably/docs#690]
See https://repl.it/

It's a pretty impressive tool, allowing you to run code in pretty much every language, even if that code runs continuously (i.e. evented Node.js code), and even allows code iFrame embedding.

I set up an EventStream example: https://repl.it/repls/PhonyAjarGame

┆Issue is synchronized with this [Jira Task](https://ably.atlassian.net/browse/EDU-1771) by [Unito](https://www.unito.io)

**Comment by mattheworiordan** [comment]
I have asked if they provide an API, not answer yet.

**Comment by mattheworiordan** [comment]
https://codesandbox.io/docs/api, is an interesting project too, and it has an API

**Comment by fliptopbox** [comment]
yeah, it is cool, i've been using the IDE for a few years now, but not
tried the API. nice find Matto.

**Comment by fliptopbox** [comment]
i am pretty sure i shared this with you, or, maybe not.
https://codesandbox.io/s/ably-channel-inspector-iwu5i

it is WIP, and I'd like to pursue it, but don't have the time ATM
